weak_supervision/models/__init__.py

```diff
@@ -1,3 +1,4 @@
 from weak_supervision.models.latent_alignment import LatentAlignment
+from weak_supervision.models.latent_alignment import LatentAlignmentSharedChar
```

Comment:
Looks like this is why the checks are failing. `latent_alignment` -> `latent_alignment_shared_char`.

Reply:
I made this change, but the checks still seem to fail hmm